### ISSUE TITLE
Run manual accessibility test

### ISSUE THREAD

**Issue by eliotjordan** [pulibrary/dpul#1191]
1. Run using Google Lighthouse (instead of WAVE Evaluation browser plugin, which wasn't working on dpul).
1. Make tickets for resolving violations.

Maybe shoot for WCAG 2.1 AA, as per https://github.com/pulibrary/pulfalight/issues/109

Pages to test:
- [x] front page
- [x] an exhibit home page
- [x] an exhibit search results page
- [x] a show page
- [x] Other exhibit pages?

**Comment by thanyabegum** [comment]
After running Google Lighthouse on the pages mentioned above, the following issues were created: #1215, #1216, #1217, #1218, #1219, #1220, and #1221.
